src/tools/cgxget.c

```diff
@@ -861,13 +861,12 @@ int main(int argc, char *argv[])
 		cgroup_set_default_systemd_cgroup();
 
 	ret = convert_cgroups(&cg_list, cg_list_len, version, CGROUP_DISK);
-	if (ret == ECGNOVERSIONCONVERT && ignore_unmappable)
+	if (ret == ECGNOVERSIONCONVERT && !ignore_unmappable)
```

Comment:
I don't think this is correct.  We need to fail on any value of `ret` or if `ret == ECGNOVERSIONCONVERT` and `ignore_unmappable` is not set.

What do you think of this?
```
if ((ret != ECGNOVERSIONCONVERT && ret) ||
    (ret == ECGNOVERSIONCONVERT && !ignore_unmappable))
```